src/renderer.ts

```diff
@@ -769,53 +769,6 @@ const renderEditions = (format: Format, excludeStyles = false) => (
 			return h(EditionsPullquote, { quote, attribution, format, key });
 		}
 
-		case ElementKind.LiveEvent:
-			return h(LiveEventLink, { ...element, key });
-
-		case ElementKind.Tweet:
-			return h(Tweet, { content: element.content, format, key });
-
-		case ElementKind.Callout: {
-			const { campaign, description } = element;
-			return h(CalloutForm, { campaign, format, description });
-		}
-
-		case ElementKind.Embed:
-			return h(EmbedComponent, { embed: element.embed });
-
-		case ElementKind.Instagram:
-			return instagramRenderer(element);
```

Comment:
These aren't atoms FYI. In summary:

- `LiveEvent`s are related to our live and masterclasses products
- `Tweet`s are tweets
- `Callout`s are requests for user feedback - seems reasonable to remove if Editions doesn't support this
- `Embed`s are _all video, audio and generic embeds_ - this may remove a _lot_ of content including YouTube and Spotify embeds
- `Instagram` are Instagram